src/app/app.module.ts

```diff
@@ -6,23 +6,23 @@ import { AppComponent } from './app.component';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 
 import { AppRoutingModule } from './app-router.module';
-
 import { CouchService } from './shared/couchdb.service';
 import { AuthService } from './shared/auth-guard.service';
 import { UserService } from './shared/user.service';
 import { ValidatorService } from './validators/validator.service';
-
+import { MatSnackBarModule } from '@angular/material/snack-bar';

```

Comment:
The material module should be define in material file . Could you please remove this and define it in Materail file.